server/chat-plugins/trivia.ts

```diff
@@ -91,6 +99,7 @@ interface TriviaData {
 	leaderboard?: TriviaLeaderboard;
 	altLeaderboard?: TriviaLeaderboard;
 	ladder?: TriviaLadder;
+	history?: TriviaHistory;
```

Comment:
```suggestion
	history?: TriviaGame[];
```

I would not alias a type for something this simple – I think it makes it _less_ readable because now you have to wonder "how exactly _is_ TriviaHistory structured? clearly not just an array of TriviaGames or this would just be an array type".